browser/ui/views/brave_help_bubble/brave_help_bubble_delegate_view.cc

```diff
@@ -0,0 +1,209 @@
+        points[2] = {bounds_f.x() + BubbleBorder::kVisibleArrowRadius,
+                     bounds_f.y()};
+        points[3] = {bounds_f.right() - 1, bounds_f.bottom()};
+        break;
+      case views::BubbleArrowSide::kBottom:
+        points[0] = {bounds_f.right() - 1, bounds_f.y()};
+        points[1] = {bounds_f.x() + BubbleBorder::kVisibleArrowRadius,
+                     bounds_f.bottom()};
+        points[2] = {bounds_f.x() + BubbleBorder::kVisibleArrowRadius - 1,
+                     bounds_f.bottom()};
+        points[3] = {bounds_f.x(), bounds_f.y()};
+        break;
+    }
+
+    return SkPath::Polygon(points, kNumPoints, part == BubbleArrowPart::kFill);
+  }
+};
+}  // namespace
+
+BraveHelpBubbleDelegateView::BraveHelpBubbleDelegateView(
+    View* anchor_view,
+    const std::string& text)
+    : BubbleDialogDelegateView(anchor_view, BubbleBorder::Arrow::TOP_CENTER) {
+  SetButtons(ui::DIALOG_BUTTON_NONE);
+  set_shadow(BubbleBorder::Shadow::STANDARD_SHADOW);
+  set_corner_radius(10);
+  set_color(kBgColor);
+  SetLayoutManager(std::make_unique<views::BoxLayout>(
+      views::BoxLayout::Orientation::kVertical));
+
+  views::Label* blocked_trackers_label =
+      AddChildView(std::make_unique<views::Label>());
+  blocked_trackers_label->SetBorder(
+      views::CreateEmptyBorder(gfx::Insets::TLBR(10, 0, 8, 0)));
+  SetUpLabel(blocked_trackers_label, base::UTF8ToUTF16(text), 16,
+             gfx::Font::Weight::SEMIBOLD);
+
+  views::Label* view_label = AddChildView(std::make_unique<views::Label>());
+  SetUpLabel(view_label,
+             l10n_util::GetStringUTF16(
+                 IDS_BRAVE_SHIELDS_ONBOARDING_CLICK_TO_VIEW_LABEL),
+             14, gfx::Font::Weight::NORMAL);
+
+  AddChildView(view_label);
+}
+
+BraveHelpBubbleDelegateView::~BraveHelpBubbleDelegateView() = default;
+
+std::unique_ptr<NonClientFrameView>
+BraveHelpBubbleDelegateView::CreateNonClientFrameView(views::Widget* widget) {
+  std::unique_ptr<NonClientFrameView> frame =
+      BubbleDialogDelegateView::CreateNonClientFrameView(widget);
+  CHECK(frame);
+
+  std::unique_ptr<BorderWithArrow> border =
+      std::make_unique<BorderWithArrow>(arrow(), color());
+  border->SetColor(color());
+
+  if (GetParams().round_corners) {
+    border->SetCornerRadius(GetCornerRadius());
+  }
+
+  static_cast<BubbleFrameView*>(frame.get())
+      ->SetBubbleBorder(std::move(border));
+  return frame;
+}
+
+void BraveHelpBubbleDelegateView::SetUpLabel(views::Label* label,
+                                             const std::u16string& text,
+                                             int font_size,
+                                             gfx::Font::Weight font_weight) {
+  label->SetMultiLine(true);
+  label->SetMaximumWidth(390);
+  label->SetText(text);
+  label->SetAutoColorReadabilityEnabled(false);
+  label->SetEnabledColor(SK_ColorWHITE);
+
+  gfx::FontList font_list({"Poppins", "Arial"}, gfx::Font::NORMAL, font_size,
```

Comment:
Will this ever show Poppins - do we embed the font somehow?

Reply:
AFAIK, these specified font will be applied when system has that font.
Otherwise, maybe system's default font will be used.

Reply:
Then I think we've got a problem to solve, along with design (@aguscruiz) - either we need to somehow embed the Poppins font for native UI, or we need to use the regular system font. I'm not sure but I don't think supplying "Arial" is a good choice. Almost 0 users will have Poppins installed locally.